sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **New Features**
 
 * Updated `forward_to` and `forward_dead_lettered_messages_to` parameters in `create_queue`, `update_queue`, `create_subscription`, and `update_subscription` methods on sync and async `ServiceBusAdministrationClient` to accept entities as well, rather than only full paths. In the case that an entity is passed in, it is assumed that the entity exists within the same namespace used for constructing the `ServiceBusAdministrationClient`.
+* Added support for using `~azure.core.credentials.AzureSasCredential` as credential for authenticating the clients.
```

Comment:
haha, we should have a new entry # 7.2.0b1 to hold this new feature

Reply:
this the only feature in b1? :)

Reply:
nah, I have a pending sending amqp annotated message PR to be merged: https://github.com/Azure/azure-sdk-for-python/pull/17650

but I'll let yours get merged first and then start resolving conflicts if needed.